src/Test/config.jl

```diff
@@ -2,6 +2,7 @@ struct TestConfig{T<:Real}
     atol::T # absolute tolerance for ...
     rtol::T # relative tolerance for ...
     solve::Bool # optimize and test result
+    query_number_of_constraints::Bool # can get `MOI.NumberOfConstraints` attribute
```

Comment:
This should be a separate PR.

Reply:
Done in https://github.com/jump-dev/MathOptInterface.jl/pull/1327